test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -672,6 +672,58 @@ describe('Dropdown', () => {
         .at(options.length - 1)
         .should.have.prop('selected', true)
     })
+    it('does not move up on arrow up when first item is selected when open when \'wrapSelection\' is on', () => {
+      wrapperMount(<Dropdown options={options} selection wrapSelection />)
+
+      // open
+      wrapper
+        .simulate('click')
+        .find('DropdownItem')
+        .first()
+        .should.have.prop('selected', true)
+
+      // arrow up
+      domEvent.keyDown(document, { key: 'ArrowUp' })
+      wrapper.update()
+
+      // selection should not move to last item
+      // should keep on first instead
+      wrapper
+        .find('DropdownItem')
+        .first()
+        .should.have.prop('selected', true)
+      wrapper
+        .find('DropdownItem')
+        .at(options.length - 1)
+        .should.have.prop('selected', false)
+    })
+    it('does not move down on arrow down when last item is selected when open when \'wrapSelection\' is on', () => {
```

Comment:
We try to organize our tests by prop name.  Let's add a new `describe('wrapSelection')` block.  It helps find tests, ensure coverage, and prevents duplication in test `it()` descriptions.